app/models/representation_order.rb

```diff
@@ -23,10 +23,6 @@ class RepresentationOrder < ApplicationRecord
     self.maat_reference = nil if case_type&.requires_maat_reference?.eql?(false)
   end
 
-  acts_as_gov_uk_date :representation_order_date,
-                      validate_if: :perform_validation?,
-                      error_clash_behaviour: :override_with_gov_uk_date_field_error
-
```

Comment:
This is related to the `defendants` form_step, is this required for `supporting_evidence`?